stack/global/versions.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+    aws = {
+      source                = "hashicorp/aws"
+      version               = ">= 4.0"
+      configuration_aliases = [aws.us-east-1]
```

Comment:
why do we need this?

Reply:
The billing alarm can only be setup in us-east-1 region.
I should add a comment for this.